packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -0,0 +1,92 @@
+import { ScheduledSlackNotification } from '@lightdash/common';
+import { nanoid } from 'nanoid';
+import { slackClient } from '../clients/clients';
+import { unfurlChartAndDashboard } from '../clients/Slack/SlackUnfurl';
+import { lightdashConfig } from '../config/lightdashConfig';
+import Logger from '../logger';
+import { schedulerService, unfurlService } from '../services/services';
+import { LightdashPage, Unfurl } from '../services/UnfurlService/UnfurlService';
+
+const getChartOrDashboard = async (
+    chartUuid: string | null,
+    dashboardUuid: string | null,
```

Comment:
would be nice to have resourceUuid, resourceType args here instead of spreading. below statement can be case/switch-ed with `assertUnreachable` in the end